src/renderer/components/Breadcrumb/NFTCrumb.js

```diff
@@ -109,4 +113,7 @@ export default function NFTCrumb() {
       ) : null}
     </>
   );
-}
+};
+
+// $FlowFixMe
```

Comment:
Why?

Reply:
Shouldn't it be `memo<{}>(NFTCrumb)` to avoid the flow fix?

Reply:
I added $FlowFixMe because when I add props to memo, I end up with the lint on save changing it to something that doesn't work. 

https://user-images.githubusercontent.com/44363395/161539842-77d0a777-1278-4fb9-bb07-00ff99f534c8.mov

Is it only doing that with me ?

Reply:
Not happening to me for sure

Reply:
Ok it's fixed now 👌